Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -16,6 +16,9 @@ public function test_it_is_initializable()
         );
     }
 
+    /**
+     * @group legacy
+     */
     public function test_with_default_configuration()
```

Comment:
Is it possible to rename this test with something like `test_with_old_default_configuration` (or something better, I'm not very inspired) and create a new one to test the new default configuration (without any deprecated key) generated by symfony?